tests/ttnn/unit_tests/gtests/accessor/kernels/accessor_benchmark.cpp

```diff
@@ -12,11 +12,29 @@ void kernel_main() {
     constexpr uint32_t page_size = get_compile_time_arg_val(1);
     constexpr uint32_t rank = get_compile_time_arg_val(2);
     constexpr uint32_t num_banks = get_compile_time_arg_val(3);
-    constexpr uint32_t base_idx = 4;
+    constexpr uint32_t base_idx_cta = 4;
 
-    using input_dspec = distribution_spec_t<base_idx, rank, num_banks>;
+    using input_dspec_cta = distribution_spec_t<base_idx_cta, rank, num_banks>;
+    // runtime tensor shape, runtime shard shape, and static bank coords
+    constexpr uint32_t base_idx_cta_DDS = base_idx_cta + compile_time_args_skip<input_dspec_cta>;
+    constexpr uint32_t base_idx_rta_DDS = 0;
+    using input_dspec_crta_DDS = distribution_spec_t<base_idx_cta_DDS, rank, num_banks>;
```

Comment:
```suggestion
    using input_dspec_crta_DDS = distribution_spec_t<base_idx_crta_DDS, rank, num_banks>;
```
Why do we need `base_idx_cta_DDS` here?
it is too difficult to understand for me :(

Reply:
In the benchmark I push arguments for four variations of arguments, and I have to move base index for cta and rta for each of them. 

Also, I don't quite understand suggested change :) You don't like variables naming?

Reply:
> In the benchmark I push arguments for four variations of arguments, and I have to move base index for cta and rta for each of them.
>Also, I don't quite understand suggested change :) You don't like variables naming?

Maybe just naming problem.
When looking at the code without full context, it's hard to tell whether it's a bug or intended behavior—since `crta` is on the left and `cta` is on the right.

Is this really an example testing **CRTA = common runtime argument**?